modular_skyrat/modules/bitrunning/code/virtual_domains/ancient_milsim/outfit.dm

```diff
@@ -0,0 +1,75 @@
+/datum/outfit/solfed_bitrun
+	name = "Bitrunning SolFed Marine"
+
+	uniform = /obj/item/clothing/under/rank/security/officer/hecu
+	head = null
+	mask = /obj/item/clothing/mask/gas/hecu2
+	gloves = /obj/item/clothing/gloves/combat
+	suit = null
+	shoes = /obj/item/clothing/shoes/combat
+	back = /obj/item/storage/backpack/ert/odst/hecu
+	glasses = null
+	ears = /obj/item/radio/headset/headset_faction/bowman
+	id = /obj/item/card/id/advanced/solfed
+	r_hand = null
+	l_hand = null
+	backpack_contents = null
+	belt = null
+	id_trim = /datum/id_trim/solfed_bitrun
+
+/datum/id_trim/solfed_bitrun
+	trim_icon = 'modular_skyrat/master_files/icons/obj/card.dmi'
+	assignment = "SolFed Marine"
+	trim_state = "trim_solfed"
+	department_color = COLOR_SOLFED_GOLD
+	subdepartment_color = COLOR_SOLFED_GOLD
+	sechud_icon_state = SECHUD_SOLFED
+
+/datum/outfit/cin_soldier_corpse
+	name = "Coalition Operative Corpse"
+	uniform = /obj/item/clothing/under/syndicate/rus_army/cin_surplus/forest
```

Comment:
These should use the random color variant just so the soldier don't all look identical, it'd improve the look a bit.

Reply:
Tarkov LARP.